### DISCUSSION TITLE
Unable to Instantiate RegexpSinglelineJavaCheck Class in TreeWalker Module

### DISCUSSION THREAD

**Discussion by Anmol202005** [checkstyle/checkstyle#16056] [Q&A]
I'm encountering an issue with Checkstyle configuration while trying to run the checks :
![image](https://github.com/user-attachments/assets/e8c07a5d-a46d-4f69-9a10-77155e81a221)

I have checked that RegexpSinglelineJavaCheck is correctly specified in the configurational file 

![image](https://github.com/user-attachments/assets/545d11c6-9f0e-43f5-8cbf-f31f4ba21d76)

**Comment by romani** [comment]
Looks like you are doing issue on activation of Examples.
Please create PR with what you have and it will share all details that I need to help you

**Reply by Anmol202005** [reply]
I have created a PR with the changes i made.
